src/datasets/infra/repositories/DatasetsRepository.ts

```diff
@@ -7,6 +7,9 @@ import { transformDatasetUserPermissionsResponseToDatasetUserPermissions } from
 import { DatasetLock } from '../../domain/models/DatasetLock';
 import { transformDatasetLocksResponseToDatasetLocks } from './transformers/datasetLocksTransformers';
 
+export interface GetCitationQueryParams {
+    includeDeaccessioned: boolean;
+}
```

Comment:
The rest of the query params used in the methods don't have an interface. I think we should be consistent, either all of them have an explicit type declaration, or they don't

Reply:
agreed, I removed this.